ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml

```diff
@@ -127,6 +138,48 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-azure
   timeout: 5h0m0s
+- always_run: false
+  as: control-plane-multi-ipsec-3nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D4ps_v5
+      ADDITIONAL_WORKERS: "1"
+      BASE_DOMAIN: ci.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: Standard_D4s_v3
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_D4s_v3
+      OCP_ARCH: amd64
+    test:
+    - ref: ipi-install-heterogeneous
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-azure-ipi-ovn-ipsec
+  timeout: 5h0m0s
+- always_run: false
+  as: control-plane-multi-ipsec-120nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D4ps_v5
+      ADDITIONAL_WORKERS: "60"
+      BASE_DOMAIN: ci.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "60"
+      COMPUTE_NODE_TYPE: Standard_D4s_v3
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_D4s_v3
+      OCP_ARCH: amd64
+    test:
```

Comment:
```suggestion
    test:
      - ref: openshift-qe-installer-azure-update-allocated-outbound-ports
```

Reply:
This is from https://github.com/openshift/release/pull/55867, wait for that pr to be merged and test it here.